pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/barnepensjon/migrering/EnkeltVedtakOmregningNyttRegelverk.kt

```diff
@@ -99,75 +103,77 @@ object EnkeltVedtakOmregningNyttRegelverk : EtterlatteTemplate<BarnepensjonOmreg
+                    Language.Bokmal to "Hvis du mener vedtaket er feil, kan du klage innen seks uker fra 1. januar 2024. Klagen skal være skriftlig. Du finner skjema og informasjon på ${Constants.KLAGE_URL}.",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "If you believe the decision is incorrect, you can appeal within six weeks after 1 January 2024. The appeal must be in writing. You can find the form and information online: ${Constants.Engelsk.KLAGE_URL}."
                 )
             }
 
             title2 {
                 text(
                     Language.Bokmal to "Du må melde fra om endringer",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "You must report any changes"
                 )
             }
             paragraph {
                 text(
                     Language.Bokmal to "Du har plikt til å melde fra til oss om endringer som har betydning for utbetalingen av barnepensjon, eller retten til å få barnepensjon. I vedlegget «Dine rettigheter og plikter» ser du hvilke endringer du må si fra om.",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "You are obligated to notify us of any changes that affect the payment of a children's pension, or the right to receive a children's pension. You will see which changes you must report in the attachment, Your Rights and Obligations."
                 )
             }
 
             title2 {
                 text(
                     Language.Bokmal to "Har du spørsmål?",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "Any questions?"
                 )
             }
             paragraph {
                 text(
-                    Language.Bokmal to "Du finner mer informasjon på nav.no/barnepensjon. Hvis du ikke finner svar på spørsmålet ditt, kan du ringe oss på telefon 55 55 33 34 hverdager 9-15. Om du oppgir fødselsnummer til barnet, kan vi lettere gi deg rask og god hjelp.",
+                    Language.Bokmal to "Du finner mer informasjon på ${Constants.BARNEPENSJON_URL}. Hvis du ikke finner svar på spørsmålet ditt, kan du ringe oss på telefon ${Constants.KONTAKTTELEFON_PENSJON} hverdager 9-15. Om du oppgir fødselsnummer til barnet, kan vi lettere gi deg rask og god hjelp.",
                     Language.Nynorsk to "",
-                    Language.English to ""
+                    Language.English to "For more information, visit us online: ${Constants.Engelsk.BARNEPENSJON_URL}. If you cannot find the answer to your question, you can call us by phone (${Constants.KONTAKTTELEFON_PENSJON}) weekdays 9-15. If you provide your child's national identity number, we can more easily provide you with quick and good help."
```

Comment:
"If you provide your child's national identity number, we can more easily provide you with quick and good help."
->
"In order to help us with a more rapid response, please provide your child's national identity number."